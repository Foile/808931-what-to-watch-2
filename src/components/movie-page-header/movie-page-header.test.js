import React from 'react';
import renderer from "react-test-renderer";
import MoviePageHeader from './movie-page-header';

it(`MoviePageHeader correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<MoviePageHeader
    auth = {{name: `test`, avatarUrl: `ya.ru`}} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
