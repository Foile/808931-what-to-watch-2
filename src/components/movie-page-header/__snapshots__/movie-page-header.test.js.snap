// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageHeader correctly renders after relaunch 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <div
      className="user-block__avatar"
    >
      <img
        alt="test"
        height="63"
        src="https://htmlacademy-react-2.appspot.com/ya.ru"
        width="63"
      />
    </div>
  </div>
</header>
`;
